<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-O15LHT5K" failures="1" tests="2" name="Test" time="16.61" errors="0" timestamp="2023-10-23T22:23:37 IST">
  <testcase classname="com.ash.tests.LoginPageTest" name="login2" time="0.876">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.ash.tests.LoginPageTest.login2(LoginPageTest.java:29)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- login2 -->
  <testcase classname="com.ash.tests.LoginPageTest" name="login" time="4.352"/>
</testsuite> <!-- Test -->
